core/src/main/java/com/opensymphony/xwork2/util/ValueStack.java

```diff
@@ -28,23 +30,25 @@
  */
 public interface ValueStack {
 
-    public static final String VALUE_STACK = "com.opensymphony.xwork2.util.ValueStack.ValueStack";
+    String VALUE_STACK = "com.opensymphony.xwork2.util.ValueStack.ValueStack";
```

Comment:
Any value defined inside an interface is by default final and static - this has changed since Java 7 I think, so you don't have to explicite specify `static final` anymore :)

Reply:
Thanks for pointing out why `static final`  is redundant for interfaces in newer Java versions.  :)